bin/hdfcoinc/pycbc_make_hdf_coinc_workflow

```diff
@@ -197,7 +192,7 @@ for insp in full_insps:
     hist_summ += list(grouper(hists, 2))
 
 # Calculate the inspiral psds and make plots
-full_segs, psd_files, insp_segs, insp_data_segs = [], [], {}, {}                            
+full_segs, psd_files, insp_segs, insp_data_segs = [], [], {}, {}                      
```

Comment:
I think not intentionally. I originally though I should rewrite this line to include some other information, but decided it was simpler to leave as is and so reverted it, with apparently a whitespace change.